### PR TITLE
Update to `polkadot-v0.9.8`

### PR DESCRIPTION
**Pull request by NukeManDan**
- [x] Update tag used on parachain template once https://github.com/substrate-developer-hub/substrate-parachain-template/pull/58/files is merged
- [X] Update chainspec files

### CODE REVIEW

**Comment by jimmychu0807** [comment]
As the polkadot version updated, do we need to regenerate the 3 set chainspec raw and plain files?

**Comment by NukeManDan** [comment]
No. This was based on a bad complication flag. I built these today, with the correct `--release` flag (all that is needed now)

File: en/1-prep/2-chain-spec.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ for your relay chain network. You can use one that is included with this worksho
 
 Whichever chain spec file you choose to use we will refer to the file simply as `chain-spec.json`
 in the instructions below. You will need to supply the proper path to the chain spec you are using.
-These *conventionally* live in a `/res` folder that is published in your node's codebase for others to use. As an example:
+These _conventionally_ live in a `/res` folder that is published in your node's codebase for others to use. As an example:
```
Comment (jimmychu0807) [inline en/1-prep/2-chain-spec.md:12]: fyi, Using `*xxx*`, and `_xxx_` have the same effect in markdown.
More info: https://guides.github.com/features/mastering-markdown/ (search for "Emphasis")

File: en/1-prep/2-chain-spec.md [hunk 2]
```diff
@@ -33,16 +33,16 @@ local test network:
 
 <!-- for some reason these links can't be markdown. See https://github.com/substrate-developer-hub/cumulus-workshop/issues/16 -->
 
-- <a href="shared/chainspecs/rococo-local-2-raw.json">shared/chainspecs/rococo-local-2-raw.json</a>:
-A two-validator relay chain with Alice and Bob as authorities. Useful for registering a single parachain. Plain chain spec <a href="shared/chainspecs/rococo-local-2-plain.json">included</a>.
+- <a href="shared/chainspecs/rococo-custom-2-raw.json">shared/chainspecs/rococo-custom-2-raw.json</a>:
```
Comment (jimmychu0807) [inline en/1-prep/2-chain-spec.md:36]: yea, let's agree on the filename convention. I am fine starting the name with `rococo-custom-*`, then sticks to the node number and the type of file. So the chainspec filename will be like `rococo-custom-2-plain.json`, or `rococo-custom-3-raw.json`

**Comment by jimmychu0807** [comment]
I will rename some of your chainspecs filenames

**Review by jimmychu0807 — APPROVED** [review]

**Pull request merged**